test/e2e/metrics_e2e_test.go

```diff
@@ -116,6 +117,45 @@ var _ = Describe("Metrics are generated for OLM managed resources", func() {
 				})
 			})
 		})
+
+		When("the OLM pod restars", func() {
```

Comment:
```suggestion
		When("the OLM pod restarts", func() {
```

Reply:
The setup of the test feels a little unnatural to me. I would expect a `When` block inside this [`Context`](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/metrics_e2e_test.go#L121). And the layout ideally would something be: 

```
Context("Metrics emitted by objects during operator installation", func() {

  When("A CSV is created", func() {
			BeforeEach(func() {
                                   // create CSV
                       }
                      It("emits a CSV metrics", func() {
                              // check for CSV metrics
                      }
                     When("The OLM pod restarts", func() {
                         BeforeEach(func(){
                                // kill the deployment and wait for it to come up again
                         }
                        It("The csv metrics are preserved" func(){
                                 // check for CSV metrics again
                       }) 
                     })
  }
```

This leaves room for extending this portion of the test suite in the future